share/completions/usbip.fish

```diff
@@ -0,0 +1,69 @@
+function __fish_usbip_no_subcommand -d 'Test if usbip has yet to be given the subcommand'
```

Comment:
Instead of hard-coding a list of subcommands (that _might_ change over time, although for some utilities they're pretty much set in stone) I think you can just use `__fish_is_nth_token 1` as the condition, which tests that the current token being completed is the first non `-` or `--` argument to the command, e.g. `usbip TAB` would pass, `usbip --verbose TAB` would pass, but `usbip attach TAB` wouldn't.